### DISCUSSION TITLE
Feature Idea: PowerUps

### DISCUSSION THREAD

**Discussion by Bolrik** [LPGameDevs/EditarrrPublic#24] [Ideas]
For a later stage of development we could add some PowerUps to give the gameplay some variety.
Nothing too fancy though.

Maybe a Cannon to deal with enemies and/or do 'Rocket'-Jumps, break obstacles etc.
Or Maybe the Player has the Cannon by default and can collect ammo?

I think there could be a ton of Items/Gadgets that come to mind but first we should keep it simple.

**Comment by yanniboi** [comment]
Power ups are a great idea! I had the thought that we could configure a bunch of different abilities in the player controller (double jump / wall jump / jetpack / etc) but lock those from the start. Then the level designer could place "unlock pickups" in the level in order to create almost a mini metroidvania-like to gate the end from being accessible without the pickups.

**Comment by DaSeabass** [comment]
With power ups and abilities, you’ll want the other end of the equation which are varied enemy types and unique “locks”. Things like shield enemies who can only be stunned by the cannon or vertical platforms which need a wall climb to get over. I love the idea for the feature, and I think adding these as well will resolve the question “why should I add this to my level”?
